src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -2239,14 +2239,14 @@ bool entitiesQuery
     catch (const std::exception &e)
     {
       *err = e.what();
-      LM_E(("Runtime Error (exception in nextSafe(): %s)", e.what()));
+      LM_E(("Runtime Error (exception in nextSafe() at %s:%d: <%s> - query: <%s>)", __FUNCTION__, __LINE__, e.what(), query.toString().c_str()));
       releaseMongoConnection(connection);
       return false;
     }
     catch (...)
     {
       *err = "generic exception at nextSafe()";
-      LM_E(("Runtime Error (generic exception in nextSafe())"));
+      LM_E(("Runtime Error (generic exception in nextSafe() at %s:%d - query: <%s>)", __FUNCTION__, __LINE__, query.toString().c_str()));
```

Comment:
Mmmm... you are right. In the of LM_E the information is already there. I'm afraid that I extend the same solution that I used in get*Fiel() at safeMongo module to a place where it is not actually needed.

I'll fix all them.